carve.lic

```diff
@@ -41,6 +43,7 @@ class Carve
 
     wait_for_script_to_complete('buff', ['carve'])
     Flags.add('carve-assembly', 'another finished wooden (hilt|haft)', 'another finished (long|short) wooden (pole)')
+    Flags.add('magic-ready', 'You feel fully prepared to cast your spell.')
```

Comment:
💡 Flags are global. That means each of these scripts is reading the same Flag. Since you `.add` at the top, this is ok (`add` sets it to false), but just be aware.

If you wanted to be extra safe, you might add a prefix or suffic to make each Flag uniquely named.

Reply:
should they be deleted on script close then?

Reply:
unique makes sense, I'll change that just in case.

Reply:
it explains why I needed to use if Flags['magic-ready'].include?('You feel fully prepared to cast your spell.') at least, Flags['magic-ready'] was true at odd times while testing.